docker-compose.yml

```diff
@@ -398,11 +398,18 @@ services:
       # Required to mount volumes for docker-in-docker executions
       - IPL_GTFS_IMPORTER_HOST_CUSTOM_SCRIPTS_DIR=${PWD}/etc/gtfs/
       - IPL_GTFS_IMPORTER_HOST_GTFS_OUTPUT_DIR=${PWD}/var/gtfs/
+      # directory where datasets should stored for publishing (Note: should be mounted as volume)
+      - WWW_ROOT_DIR=/var/www/
+      # RadVIS Download credentials
+      - RADVIS_WFS_USER=${RADVIS_WFS_USER:?missing/empty}
+      - RADVIS_WFS_PASSWORD=${RADVIS_WFS_PASSWORD:?missing/empty}
     volumes:
       # Make docker API accessible so we can start containers in jobs
       - /var/run/docker.sock:/var/run/docker.sock
       # mount storage dir so logs/assets are written to volume shared with dagaster-daemon/dagit
       - ./var/dagster/storage/:/opt/dagster/dagster_home/storage/
+      # mount www dir to republish downloaded datasets
+      - ./var/www/datasets/:/var/www/
```

Comment:
Apart from possible ownership issues, I don't expect problems. I'll check via dev deployment of this branch

Reply:
Since `/tmp` and `var` are on different file-systems, this results in

```
OSError: [Errno 18] Invalid cross-device link: '/tmp/tmphh8x9z7u/radnetz_bw.gpkg.gz' -> '/var/www/radvis/radnetz_bw.gpkg.gz'
  File "/usr/local/lib/python3.11/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/usr/local/lib/python3.11/site-packages/dagster/_utils/__init__.py", line 465, in iterate_with_context
    next_output = next(iterator)
                  ^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 131, in _coerce_op_compute_fn_to_iterator
    result = invoke_compute_fn(
             ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 125, in invoke_compute_fn
    return fn(context, **args_to_pass) if context_arg_provided else fn(**args_to_pass)
                                                                    ^^^^^^^^^^^^^^^^^^
  File "/opt/dagster/app/pipeline/assets/radvis.py", line 33, in radnetz_bw_download
    download(RADVIS_DOWNLOAD_URL, destination_folder, RADVIS_OUT_FILENAME, create_precompressed=True)
  File "/opt/dagster/app/pipeline/util/urllib.py", line 72, in download
    os.replace(tmpfilename + '.gz', finalfilename + '.gz')
```
This requires a different download/rename handling in ipl-dagster-pipeline.